services/api-server/src/simcore_service_api_server/core/settings.py

```diff
@@ -161,11 +162,12 @@ def _enable_only_in_devel_mode(cls, v: Any, values: Mapping[str, Any]):
             if (
                 values
                 and (boot_mode := values.get("SC_BOOT_MODE"))
-                and boot_mode.is_devel_mode()
+                and not boot_mode.is_devel_mode()
             ):
-                raise ValueError(
+                error_message = (
```

Comment:
Same here, suggestion by ruff to put string first in variable. I assume this is avoid crashing inside the ValueError in case something is weird with the string

Reply:
@wvangeit Rush does this so that the message does not appears twice in the log when the exception raises. There are some links to the rush codes that explains the rationale